lsp/mesonlsp.lua

```diff
@@ -3,8 +3,33 @@
 --- https://github.com/JCWasmx86/mesonlsp
 ---
 --- An unofficial, unendorsed language server for meson written in C++
+
+local meson_matcher = function(_, path)
```

Comment:
docstring should give an example of the line this is trying to find. shouldn't have to study the code to figure that out.

Reply:
I don't understand what you mean, can you explain in more detail please?

Reply:
```suggestion
--- Decides if a file is a meson file by looking for lines like:
--- "foo.bar" or "baz"
local meson_matcher = function(_, path)
```

replace "foo.bar" with an actual example

Reply:
what is an example of text content that this looks for ?

Reply:
Done